std_msgs/Header header
geometry_msgs/Pose pose
string segment_name #name of a specific component of the object
string subject_name #name of the entire object
int32 frame_number #unit of time for each capture
string translation_type #Local or Global
